alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/MonitorRunResult.kt

```diff
@@ -25,12 +25,12 @@ import org.elasticsearch.script.ScriptException
 import java.time.Instant
 
 data class MonitorRunResult(
-    val monitorName: String,
-    val periodStart: Instant,
-    val periodEnd: Instant,
-    val error: Exception? = null,
-    val inputResults: InputRunResults = InputRunResults(),
-    val triggerResults: Map<String, TriggerRunResult> = mapOf()
+        val monitorName: String,
```

Comment:
What message did you get for the code style check? The previous way of using 4 spaces was correct indentation I believe. You can refer to the bottom of the [Class header formatting](https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting) section in Kotlin's documentation.

Reply:
cannot find those anymore, i use use IDEA’s Command + Option + L to format the document, after that all code style error gone.  I could remove space manually, but usually prefer to format code use build-in tool.

Reply:
resolve for now, but feel free to reopen if our team has different practice.

Reply:
@seraphjiang  I feel this change is visible for tab / space, could you verify it is formatting using tabs instead of space.

Reply:
it is space, not tab. IDEA Editor -> code style -> Kotlin (Schema - Default IDE)
Tab Size - 4
Indent - 4
Continuation indent - 8

Reply:
@mihirsoni @qreshi 
after a little experiment, here is result
with default ide editor code style settings, we will have to format the class header manually. anyone know better trick, let me know :)